.github/actions/decode_signing_key_action/action.yml

```diff
@@ -0,0 +1,18 @@
+name: Decode signing key
+description: 'Decodes gpg key into file'
+inputs:
+  signing_key_file:
+    description: 'Signing key file'
+    required: true
+    default: ''
+  signing_file_path:
+    description: 'Signing file path'
+    required: true
+    default: ''
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        echo "${{inputs.signing_key_file}}" > ~/secretKey.gpg.b64
+        base64 -d ~/secretKey.gpg.b64 > ${{ inputs.signing_file_path }}
+      shell: bash
```

Comment:
Let's be sure to keep a newline at the end of every file

Reply:
added